options/locale/locale_en-US.ini

```diff
@@ -430,7 +430,7 @@ remove_account_link = Remove linked account
 remove_account_link_desc = Delete this account link will remove all related access for your account. Do you want to continue?
 remove_account_link_success = Account link has been removed successfully!
 
-repos_none = You do not owned any repository
+repos_none = You do not currently have any repositories
```

Comment:
All the other strings refer to it as owning repositories. Consistency makes everything easier to use and look like everything fits together. What do you mean by it isn't too accurate.